resources/assets/js/components/JobDetails/JobDetails.tsx

```diff
@@ -333,20 +334,7 @@ const JobDetails: React.FunctionComponent<
       .required(intl.formatMessage(validationMessages.required)),
     classification: Yup.mixed()
       .oneOf(
-        [
-          "AS",
-          "BI",
-          "CO",
-          "CR",
-          "CS",
-          "EC",
-          "EX",
-          "FO",
-          "IS",
-          "PC",
-          "PE",
-          "PM",
-        ],
+        Object.values(ClassificationId),
```

Comment:
To keep the rest of the code equivalent, you actually want `Object.keys(ClassficiationId)` here, not Object.values

Reply:
Fixed in 33cbd558a98e856a84c68cc61da1d5055328f4b4